app/assets/javascripts/components/generic_object/generic-object-definition-component.js

```diff
@@ -130,7 +130,9 @@ function genericObjectDefinitionFormController(API, miqService, $q) {
 
   vm.saveWithAPI = function(method, url, saveObject, saveMsg) {
     miqService.sparkleOn();
-    API[method](url, saveObject)
+    API[method](url, saveObject, {
+      skipErrors: [400],  // disable error modal on validation errors
```

Comment:
Is it possible to make this a generic fix in `miq_api.js` itself, so that you don't have to make individual fixes in all other controllers?

Reply:
Or does it look like, we need to extract `saveWithAPI` in `miqService`, so that the fix can be applied across all controllers via `miqService`?

Reply:
It looks like I may have more BZs for this, not just this one, so yes, this will need to go to other places where API validation is being used. Will do :). (In per-BZ PRs.)